support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -98,10 +99,11 @@ const days = getDisplayDays();
 
 // ----- Component ----- //
 type Plans = {
-  [WeeklyBillingPeriod]: {
+  [WeeklyBillingPeriod | typeof Monthly]: {
```

Comment:
Why does this need to accept 'Monthly'?

Reply:
Flow was chucking a hissy fit about it - I'm still not clear why it needs it.

Reply:
Ah it's because the incoming prop is of type `BillingPeriod`, I'd just change this type to `BillingPeriod` to match.